onnx/defs/schema.h

```diff
@@ -686,6 +707,16 @@ class OpSchema final {
     return all_tensor_types_with_bfloat;
   }
 
+  static const std::vector<std::string>& all_tensor_types_with_bfloat_float8() {
```

Comment:
I wonder if the sequence types need to be updated too.

Reply:
If we agree float8 are only limited to a small subset of operators, i assumed sequence would not be part of it.